arithmetic_analysis/README.md

```diff
@@ -0,0 +1,7 @@
+# Arithmetic analysis
+
+It's a branch of mathematics that deal with solving linear equations.
```

Comment:
```suggestion
Arithmetic analysis is a branch of mathematics that deals with solving linear equations.
```
Or should we call it numerical analysis?

Reply:
Now that you talk about it, I believe numerical analysis is better:

> Examples of numerical analysis include: ordinary differential equations as found in celestial mechanics, **numerical linear algebra** in data analysis, and stochastic differential equations and Markov chains for simulating living cells in medicine and biology.

Reply:
But then, we would have to change the name of the folder too